challenges/objects-arrays.js

```diff
@@ -59,11 +85,20 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let student = 0; student < graduates.length; student++) {
+  contactInfo.push(`${graduates[student].first_name} ${graduates[student].email}`);
+}
 console.log(contactInfo);
 
 
+
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i = 0; i < graduates.length; i++) {
+  if(graduates[i].university.search('Uni') > 0){
+    uni.push(graduates[i].university);
+  }
+}
```

Comment:
Nice work here! I like how you employed 'search' on this problem. Great logic here! Another option you can consider is includes('uni'). Your solution is perfect, just another feature to be aware of.